docs/src/lang/functions.md

```diff
@@ -156,13 +158,83 @@ In this sense, the type restrictions of Apalache are similar to those for the
 generic collections of Java and Scala.  As a result, the type checker in
 Apalache rejects the three above examples.
 
+**TLA+ functions and Python dictionaries**. As we mentioned before, TLA+
+functions are similar to maps and dictionaries in programming languages. To
+demonstrate this similarity, let us compare TLA+ functions with [Python
+dictionaries][].  Consider a TLA+ function `price` that is defined as follows:
+
+```tla
+  [ meal \in { "Schnitzel", "Gulash", "Cordon bleu" } |->
+                CASE meal = "Schnitzel" -> 18
+                  [] meal = "Gulash"    -> 11
+                  OTHER                 -> 12
```

Comment:
I think the inclusion of `OTHER` here complicates the comparison. So I suggest:

```suggestion
                  "Cordon bleu"              -> 12
```

iiuc, this is also more faithful to the python encoding.